core-bundle/src/Image/PictureFactory.php

```diff
@@ -106,6 +106,11 @@ public function create($path, $size = null, ResizeOptions $options = null): Pict
             $image = $this->imageFactory->create($path);
         }
 
+        // Support arrays in a serialized form
+        if (\is_string($size)) {
+            $size = StringUtil::deserialize($size);
+        }
```

Comment:
I think this can be simplified.
```suggestion
        $size = StringUtil::deserialize($size);
```

Reply:
:+1: changed it. 

(I wasn't sure if the 'api contract' of `StringUtil::deserialize()` includes calling with anything else than a string - e.g. a `PictureConfiguration` - but that's how we use it anyway.  )